tick/base_model/model_generalized_linear.py

```diff
@@ -15,6 +16,9 @@ class ModelGeneralizedLinear(ModelLabelsFeatures):
     fit_intercept : `bool`
         If `True`, the model uses an intercept
 
+    dtype : `string`
```

Comment:
syntax should be
```dtype : `{'float64', 'float32'}`, default='float64'```

Reply:
It's also rather an attribute as it is not given to the constructor

Reply:
except it is a string who's values should only be one of those

Reply:
It's only an attribute of the superclass